haproxy.cfg

```diff
@@ -53,11 +53,10 @@ frontend https_front
     # 2) using the postgrest hostname
     # 3) be coming in from the COA network
 
-    acl path_postgrest path_beg /knack-services /legacy-scripts /parking /road-conditions /bond-reporting
```

Comment:
We're no longer checking for the specific route name on the bastion host's internet-facing proxy. There is existing route-matching in the [postgREST proxy service](https://github.com/cityofaustin/atd-postgrest) that takes care of this. 

This enables us to add new postgREST setups without modifying the bastion host config.